grails-app/controllers/org/pih/warehouse/reporting/ReportController.groovy

```diff
@@ -188,6 +188,31 @@ class ReportController {
         Location location = Location.get(session.warehouse.id)
         def data = forecastingService.getDemandDetails(location, null)
         if (params.downloadFormat == "csv") {
+            String currentLocaleTag = LocalizationUtil.localizationService.getCurrentLocale().toLanguageTag().toUpperCase()
+            data = data.collect {
```

Comment:
I don't like this approach since this report can be very big. Is there a way to achieve it through SQL query in the `getDemandDetails` method?

Reply:
At the moment I don’t have any clue. We’ve followed this way for Alan’s reports as well (in each of this report the collect was done). I can dig it on Monday to try it out, but I don’t feel like I will be able to merge two columns in one using the sql query